app/Main.hs

```diff
@@ -223,16 +225,16 @@ projectThread config options metrics projectThreadData = do
     -- Start a worker thread to run the main event loop for the project.
     Async.async
       $ void
-      $ runStdoutLoggingT
-      $ Metrics.runLoggingMonitorT
+      $ runEff
```

Comment:
One important difference right at the start: the `runLogicEventLoop` function previously took these "runners" as arguments. In `effectful`, the recommended pattern is to write the function as requiring some set of effects, before reducing the set of effects to `IO` using `runEff $ runner1 $ runner2 $ ...`

The difference is mostly bikeshedding in my opinion.